lxd/patches.go

```diff
@@ -124,6 +124,7 @@ var patches = []patch{
 	{name: "network_acl_remove_defaults", stage: patchPostDaemonStorage, run: patchNetworkACLRemoveDefaults},
 	{name: "clustering_server_cert_trust", stage: patchPreDaemonStorage, run: patchClusteringServerCertTrust},
 	{name: "warnings_remove_empty_node", stage: patchPostDaemonStorage, run: patchRemoveWarningsWithEmptyNode},
+	{name: "cloud_init_config_keys", stage: patchPostDaemonStorage, run: patchUseCloudInitConfigKeys},
```

Comment:
So I don't think we should do that as it's going to confuse tools like Juju and others which may be reading the current config keys. All our code is designed to work with both new and old name, so there's no need to mangle the existing configs.